modules/noise/doc_classes/NoiseTexture2D.xml

```diff
@@ -4,8 +4,7 @@
 		A texture filled with noise generated by a [Noise] object.
 	</brief_description>
 	<description>
-		Uses [FastNoiseLite] or other libraries to fill the texture data of your desired size.
-		NoiseTexture2D can also generate normalmap textures.
+		Uses [FastNoiseLite] or other libraries to fill the texture data of your desired size. [NoiseTexture2D] can also generate normalmap textures.
```

Comment:
I would suggest standardizing on "normal map" and changing the couple occurrences of "normalmap":
```
$ rg -g'*.xml' "normal map" | wc -l
25
$ rg -g'*.xml' "normalmap" | wc -l
2
```